docs/remote_inference_blueprints/amazon_rekognition_connector_blueprint.md

```diff
@@ -0,0 +1,259 @@
+# Amazon Rekognition Detect Text connector blueprint
+
+Amazon Rekognition is a computer vision service that can analyze images and videos. This tutorial shows how to create an OpenSearch connector for Amazon Rekognition's Detect Text API, which can detect and extract text from images.
+
+## Detect text with DetectText API
+
+This tutorial demonstrates how to create an OpenSearch connector for an Amazon Rekognition model that detects text in images using the DetectText API.
```

Comment:
You wrote that this is for the blueprint but also added the ML Inference processor. Maybe add that here too? Other tutorials just mention how to deploy and then run inference so this important info could get lost.

Reply:
good idea. will add this part